app/assets/javascripts/exhibit_search_typeahead.js

```diff
@@ -63,9 +62,9 @@
             'No matches found',
             '</div>'
           ].join('\n'),
-          suggestion: Handlebars.compile(
-            '<div class="exhibit-result">{{title}} <span class="subtitle">{{subtitle}}</span></div>'
-          )
+          suggestion: function(suggestion) {
+            return `<div class="exhibit-result">${suggestion.title} <span class="subtitle">${suggestion.subtitle ?? "" }</span></div>`
```

Comment:
FWIW template literal syntax has been widely supported since 2015 and the nullish coalescing operator since 2020.

Reply:
Should we update the houndci config?

Reply:
I looked into updating the configuration, but houndci is lagging on supported versions for many of the linters it uses. I've created an exhibits issue (https://github.com/sul-dlss/exhibits/issues/2481) to decide whether to configure it to use eslint instead of jshint or to remove it altogether. Maybe we can discuss as a team at stand-up this week.